.buildkite/pipeline.yml

```diff
@@ -319,6 +319,73 @@ steps:
         agents:
           slurm_mem: 20GB
 
+  - group: "Milestones"
```

Comment:
This group has been removed. Please add only the topography experiments to the current groups we have.